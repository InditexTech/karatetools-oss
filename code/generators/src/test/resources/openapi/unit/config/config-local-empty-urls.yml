# urls: karate urls used for target environments and authentication
urls:
  #karate-utils-new-karate-url-marker (do not remove) - new generated apis urls will be placed here automatically

# defaultUsername: default username
# defaultAuthMode: basic | janus | jwt | openAM
# credentials: pairs of username: password
