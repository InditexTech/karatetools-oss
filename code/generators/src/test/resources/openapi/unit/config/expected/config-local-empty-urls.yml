# urls: karate urls used for target environments and authentication
urls:
  testUrl: "#('http://localhost:' + (karate.properties['APP_PORT'] || 8080) + '/TO_BE_COMPLETED')"
  #karate-utils-new-karate-url-marker (do not remove) - new generated apis urls will be placed here automatically

# defaultUsername: default username
# defaultAuthMode: basic | janus | jwt | openAM
# credentials: pairs of username: password
